codestodians/food-basket-front-end
# [Feature]: Abstract Routes

#### What does this PR do?
- Creates a collection of routes to be rendered in the App component

#### Description of Task to be completed?
- Create a `routes` directory with a single file that has the route objects.
- Prune the App component due to the routes abstraction.

#### How should this be manually tested?
- Check out into the `feature/abstract-routes-158069926` branch.
- Run the app as before. Everything should work perfectly.

#### Any background context you want to provide?
- This story was created in a bid to simplify the look of the App component and also to make registering of routes simply be by creating an extra object in the routes file whose entries are simply the props that a `Route` component will take normally.

#### What are the relevant pivotal tracker stories?
[#158069926](https://www.pivotaltracker.com/story/show/158069926)
